libs/input-mapping/src/Table/Options/InputTableOptions.php

```diff
@@ -152,9 +152,16 @@ public function getStorageApiLoadOptions(InputTableStateList $states): array
         }
         if (!empty($this->definition['changed_since'])) {
             if ($this->definition['changed_since'] === self::ADAPTIVE_INPUT_MAPPING_VALUE) {
-                throw new InvalidInputException(
-                    'Adaptive input mapping is not supported on input mapping to workspace.',
-                );
+                try {
+                    $lastImportDateString = $states
+                        ->getTable($this->getSource())
+                        ->getLastImportDate();
+
+                    // converting to unix timestamp https://keboolaglobal.slack.com/archives/C054VSPFVST/p1723555870048739?thread_ts=1723531121.814779&cid=C054VSPFVST
+                    $exportOptions['changedSince'] = strtotime($lastImportDateString);
```

Comment:
Tady bylo potreba `strtotime`. 

Pro ten S3 export se to nekde dela taky, ale nenasel jsem presne kde se to deje.